extensions/packageManager/src/node/index.ts

```diff
@@ -28,7 +28,7 @@ const hasSchema = (c) => {
 };
 
 const isAdaptiveComponent = (c) => {
-  return hasSchema(c) || c.includesExports;
+  return c.name && c.version && (hasSchema(c) || c.includesExports);
```

Comment:
Nit - you could do c?.name to handle c being undefined since c doesn't have any type, but since this is internal you can protect it from the caller.